sources/core/Stride.Core/build/Stride.Core.targets

```diff
@@ -72,15 +72,15 @@
       <StrideAddReference Include="@(ReferencePath)" Condition="'%(ReferencePath.NuGetPackageId)' == ''"/>
     </ItemGroup>
     <PropertyGroup>
-      <StrideAssemblyProcessorOptions Condition="'$(StrideAssemblyProcessorOptions)' == ''">--auto-notify-property --parameter-key --auto-module-initializer --serialization</StrideAssemblyProcessorOptions>
+      <StrideAssemblyProcessorOptions Condition="'$(StrideAssemblyProcessorOptions)' == ''"> --parameter-key --auto-module-initializer --serialization</StrideAssemblyProcessorOptions>
```

Comment:
Nothing important, but the space before `--parameter-key` 😁

Reply:
Here you go 😁